src/schema/types/addUserMutation.graphql

```diff
@@ -1,3 +1,3 @@
-type Mutation {
+extend type Mutation {
   addUser(username:String!, firstName:String!, lastName:String!, password:String!):User!
```

Comment:
Add some air here after `:`.

Reply:
@nahudalla Here It'd be better if we define a new `input` to manage these input values, something like:

```
input UserInput {
  username: String!
  firstName: String!
  lastName: String!
  password: String!
}
```

Reply:
This mutation was created as part of the Block 3. I was thinking that maybe it's not needed anymore since now we're using the SignUp mutation to handle creating a new user. I will remove it if Abel can confirm this, or refactor the parameters as suggested by Mauricio if it's going to be needed later.

Reply:
Yeah, I guess we can remove it. @mpautasso what do you say?

Reply:
ok, If we don't need it, let's remove it.